api/BUILD

```diff
@@ -89,13 +99,16 @@ proto_library(
         "//envoy/config/rbac/v3alpha:pkg",
         "//envoy/config/resource_monitor/fixed_heap/v2alpha:pkg",
         "//envoy/config/resource_monitor/injected_resource/v2alpha:pkg",
+        "//envoy/config/retry/omit_canary_hosts:pkg",
+        "//envoy/config/retry/previous_hosts:pkg",
```

Comment:
Can you switch these to be in versioned packages? I'm working on a PR to fix `previous_priorities` as a versionless special case, trying to avoid any new ones creeping in.

Reply:
Done.